build.sbt

```diff
@@ -165,3 +260,90 @@ addCommandAlias(
   "checkScalaFormat",
   "scalafmtCheckAll ; scalafmtSbtCheck"
 )
+
+// Bootstrapping task that wires the build to the
+// codegen module's main method
+def runCodegen(config: Configuration) = Def.task {
```

Comment:
This is the build task that wires invokes the code-generator via shell-out. I'm not sure the caching was necessary in the scope of this project, but, I've just copied/pasted/adapted what we do in smithy4s.